### PR TITLE
Add MiscellaneousScreen (Using Rescript!)

### PR DESCRIPTION
**Pull request by mk-emeks**
As a proof of concept I will try to use rescript in this PR.

### Related Issue
https://github.com/emeks-studio/sensum-mobile/issues/23

### CODE REVIEW

File: src/assets/styles/Themes.res [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+
+    Belt.List.reduce(themes, stylesByThemeId, (acc, theme) =>
+      Belt.Map.String.set(acc, theme.id, StyleSheet.create(styleByThemeFn(theme)))
+    )
+  }
+
+  let unsafeGet = (stylesByThemeId: Belt.Map.String.t<Js.t<'a>>, themeId: string): Js.t<'a> =>
+    Belt.Map.String.getExn(stylesByThemeId, themeId)
+    
+  let unsafeCreate = (~themeId: string, ~styleByThemeFn: theme => Js.t<'a>): Js.t<'a> => {
```
Comment (mk-emeks) [inline src/assets/styles/Themes.res:125]: contextual types could help ?)

File: README.md [hunk 2]
```diff
@@ -38,6 +38,18 @@ nvm use
 npm install
 ```
 
+### Development
```
Comment (mirko-plowtech) [inline README.md:41]: improve this section. Add some additional notes about rescript development flow!

File: src/components/MiscellaneousScreen.res [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+      <TouchableOpacity onPress={_ => showNetwork()->ignore}>
+        <Text style={styles["text"]}>
+          {"Contar electrones que orbitan actualmente!"->React.string}
+        </Text>
+      </TouchableOpacity>
+      <Text style={styles["text"]}> {"Cambiar estilo de la app!"->React.string} </Text>
+    </View>
+    <View style={styles["footer"]}>
+      <TouchableOpacity onPress={_ => goOutside("https://emeks.com.ar")->ignore}>
+        <Text style={styles["text"]}> {"Powered by emeks"->React.string} </Text>
```
Comment (mirko-plowtech) [inline src/components/MiscellaneousScreen.res:100]: impulsado por **emeks**

File: src/components/MiscellaneousScreen.res [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+        onPress={_ =>
+          goOutside("https://github.com/emeks-studio/sensum-mobile#sensum-mobile")->ignore}>
+        <CloseIcon theming />
+      </TouchableOpacity>
+    </View>
+    <View style={styles["body"]}>
+      <TouchableOpacity
+        onPress={_ =>
+          goOutside("https://github.com/emeks-studio/sensum-mobile#sensum-mobile")->ignore}>
+        <Text style={styles["text"]}> {"Official repository"->React.string} </Text>
```
Comment (mirko-plowtech) [inline src/components/MiscellaneousScreen.res:86]: Ir al repositorio oficial

File: src/components/MiscellaneousScreen.res [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+      </TouchableOpacity>
+      <TouchableOpacity onPress={_ => goOutside("https://emeks.gitlab.io/sensum/lore/")->ignore}>
+        <Text style={styles["text"]}> {"sensum lore"->React.string} </Text>
```
Comment (mirko-plowtech) [inline src/components/MiscellaneousScreen.res:89]: Ir al lore

File: src/components/MiscellaneousScreen.res [hunk 6]
```diff
@@ -0,0 +1,104 @@
...
+      </TouchableOpacity>
+      <TouchableOpacity onPress={_ => showNetwork()->ignore}>
+        <Text style={styles["text"]}>
+          {"Contar electrones que orbitan actualmente!"->React.string}
```
Comment (mirko-plowtech) [inline src/components/MiscellaneousScreen.res:93]: Contar cantidad de electrones orbitando

File: src/components/ui/CloseIcon.res [hunk 7]
```diff
@@ -0,0 +1,5 @@
+
+// https://reason-react-native.github.io/en/docs/migration/jsx3/#notes-about-reasonreactwrapjsforreason--reasonreactwrapreasonforjs
+@bs.module("./CloseIcon") @react.component external make: (~theming: Themes.theming) => React.element = "CloseIcon"
```
Comment (mirko-plowtech) [inline src/components/ui/CloseIcon.res:3]: try to just import asset svg!

**Pull request merged**